packages/expo-error-recovery/ios/EXErrorRecovery/EXErrorRecoveryModule.h

```diff
@@ -0,0 +1,14 @@
+// Copyright © 2018 650 Industries. All rights reserved.
+
+#import <UMCore/UMExportedModule.h>
+#import <UMCore/UMModuleRegistryConsumer.h>
+
+static NSString * const USER_DEFAULTS_KEY = @"EX.expo.errorRecovery";
```

Comment:
How about leaving this without that, for a third person, strange `EX.` prefix and moving that prefix to scoped module?

Reply:
To make the transition easier, `EXErrorRecoveryModule` could expose a method returning the key for overriding.